### DISCUSSION TITLE
Why no lines number when I edit Java Files ?

### DISCUSSION THREAD

**Discussion by Web-Manager** [apache/netbeans#5377] [Q&A]
All in the title.

I use Netbeans 16 and Windows 10.

Thanks so lot for your help.

**Comment by mbien** [comment]
`view -> show line numbers` should turn them back on

**Comment by Web-Manager** [comment]
Hello, thank you for bringing me this answer, but I have already tested all the solutions found in my research for days.  Nothing works.  Only Java files do not show line numbers.  Without solutions, I have to change IDE.

**Reply by mbien** [reply]
in this case something else is going on
 - did you import (very) old configuration on first start?
 - possibly a plugin causing an issue
 - are you running NetBeans on the latest JDK update of ~11,~ 17, 21 or later

the first two issues can be solved by resetting the config by removing the config folder, see https://github.com/apache/netbeans#log-config-and-cache-locations

attaching the startup log could also help to find the cause

**Reply by Web-Manager** [reply]
Hello, 
thank you for your suggestions. 
I will try these solutions. 
Good for you.

**Comment by troizet** [comment]
Here is a solution to a similar problem https://github.com/apache/netbeans/issues/3883. Maybe this will help?

**Comment by gilbertoca** [comment]
I'm having the same problem with NetBeans 22. I've tested it using jdk 17 and 21, but no success.
Netbeans 21 + jdk11 is working.

**Reply by mbien** [reply]
https://github.com/apache/netbeans/discussions/5377#discussioncomment-4842669

**Reply by gilbertoca** [reply]
@mbien thank you! Removing $USER/.netbeans/22 and start again, fixes it.
